AEPMessaging/Sources/LiveActivity/AEPLiveActivityData.swift

```diff
@@ -0,0 +1,45 @@
+/*
+ Copyright 2025 Adobe. All rights reserved.
+ This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License. You may obtain a copy
+ of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software distributed under
+ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ OF ANY KIND, either express or implied. See the License for the specific language
+ governing permissions and limitations under the License.
+ */
+
+import Foundation
+
+/// Encapsulates data for Adobe Experience Platform integration with iOS Live Activities.
+///
+/// This struct provides the necessary identifiers and data for both managing and tracking Live
+/// Activities through Adobe Experience Platform. Use this struct when implementing the
+/// `AEPLiveActivityAttributes` protocol.
+@available(iOS 16.1, *)
+public struct AEPLiveActivityData: Codable {
```

Comment:
same here with the name - maybe just `LiveActivityData` or if you want to brand it `AdobeLiveActivityData`

Reply:
and exposing it to objc

Reply:
Same here, We can't provide objC support for live activity classes